config/forms/2021_2022.json

```diff
@@ -85,20 +105,20 @@
                   "hint_text": "",
                   "type": "radio",
                   "answer_options": {
-                    "0": {
-                      "value": "Social rent"
-                    },
                     "1": {
                       "value": "Affordable rent"
                     },
                     "2": {
-                      "value": "London affordable rent"
+                      "value": "London Affordable Rent"
+                    },
+                    "4": {
+                      "value": "London Living Rent"
                     },
                     "3": {
-                      "value": "Rent to buy"
+                      "value": "Rent to Buy"
```

Comment:
Isn't this inconsistent with how we're capitalising things elsewhere? I thought it was supposed to be regular sentence case everywhere? @Dan13L3

Reply:
@baarkerlounger In most cases it should be sentence case, yes - but we should capitalise any specific government schemes like this

Reply:
This is the name of a scheme, so I believe.